File: pset1/mario_less.c
//This program prints out a half-pyramid of a specified height.

#include <cs50.h>
#include <stdio.h>


// prompts user for height
// only accepts valid heights
int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n<0 || n>23);


// makes pyramid
for (int i = 1; i<n+1; i++)
    {
    for (int j = 0; j<n-i; j++)
    {
        printf(" ");
    }
    for (int k = 0; k < i+1; k++)
    {
        printf("#");
    }
    printf("\n");
    }
}

File: pset6/mario_less.py
from cs50 import get_int

def main():

    """Get a positive integer from the user"""
    while True:
        print("Height: ", end = "")
        n = get_int()
        if n >= 0 and n <= 23:
            break

    """Print out half pyramid"""
    for i in range (n):
        for j in range(n - i - 1):
            print(" ", end = "")
        for k in range(i + 2):
            print("#", end = "")
        print("")

if __name__ == "__main__":
    main()
